adb_shell/hidden_helpers.py

```diff
@@ -153,14 +153,16 @@ class _FileSyncTransactionInfo(object):  # pylint: disable=too-few-public-method
         The index in ``recv_buffer`` that will be the start of the next data packet sent
 
     """
-    def __init__(self, recv_message_format):
-        self.send_buffer = bytearray(constants.MAX_ADB_DATA)
+    def __init__(self, recv_message_format, maxdata=constants.MAX_ADB_DATA):
```

Comment:
Please document the `maxdata` parameter and the `_maxdata` attribute above.  (You can use the same description for both.)